spec/index.html

```diff
@@ -1528,34 +1549,14 @@ <h4>Patterns of RDFS entailment (Informative)</h4>
             <td class="othertable">xxx <code>rdf:type rdfs:Datatype .</code></td>
             <td class="othertable">xxx <code>rdfs:subClassOf rdfs:Literal .</code></td>
           </tr>
-        </tbody>
-      </table>
-      
-      <p/>
-      
-      <table id="rdfs_entailment_patterns_recursive">
-        <tbody>
-          <tr>
-            <th ></th>
-            <th >If the triple <a>appears in</a> S:</th>
-            <th >then S RDFS <a>entails recognizing D</a>:</th>
-          </tr>
-          <tr >
-            <td class="othertable"><dfn>rdfs4</dfn></td>
-            <td class="othertable">xxx aaa yyy <code>.</code></td>
-            <td class="othertable">xxx <code>rdf:type rdfs:Resource .</code><br/>
-            yyy <code>rdf:type rdfs:Resource .</code></td>
-          </tr>
           <tr >
             <td class="othertable"><dfn>rdfs14</dfn></td>
-            <td class="othertable">xxx yyy &lt;&lt;(aaa bbb ccc)>> <code>.</code></td>
-            <td class="othertable">the graph S with the appearance replaced with<br/>
-            xxx yyy _:nnn <code></code><br/><br/>
+            <td class="othertable">ttt and  &lt;&lt;(aaa bbb ccc)&gt;&gt; appears in ttt</td>
```

Comment:
```suggestion
            <td class="othertable">Any triple ttt such that  &lt;&lt;(aaa bbb ccc)&gt;&gt; appears in ttt</td>
```

Reply:
`such that  &lt;&lt;(aaa` has an extra space that ought to be removed, but will cause no trouble if not.